dcpy/connectors/socrata/metadata.py

```diff
@@ -77,7 +78,7 @@ def make_dcp_metadata(socrata_md: pub.Socrata.Responses.Metadata) -> models.Meta
         summary=socrata_md["description"],
         description=socrata_md["description"],
         tags=socrata_md["tags"],
-        each_row_is_a=socrata_md["metadata"]["rowLabel"],
+        each_row_is_a=socrata_md["metadata"].get("rowLabel") or "<FILL_ME_IN>",
```

Comment:
`<FILL_ME_IN>` <-- small nit: perhaps make it a constant to be used across the file?

Reply:
That's a great idea. We could also reuse that variable to raise an error when we deserialize the metadata and we see that it contains our `<FILL_ME_IN>` placeholder.